README.md

```diff
@@ -1191,6 +1192,14 @@ messagingSession.addObserver(observer);
 messagingSession.start();
 ```
 
+### Amazon Voice Focus and Echo Reduction
+
+Amazon Voice Focus is a noise suppressor that uses machine learning to reduce unwanted background noise in your users’ microphone input. Unlike conventional noise suppressors, Amazon Voice Focus reduces fan noise, road noise, typing, rustling paper, lawnmowers, barking dogs, and other kinds of non-speech input, allowing your users to focus on the human voice. Amazon Voice Focus offers multiple complexity levels, allowing you to trade some quality to support a wider range of devices. The Amazon Chime SDK will by default automatically choose the right complexity level at runtime based on device performance.
```

Comment:
`default automatically` sounds redundant. One of the words will be fine.

Reply:
`default automatically` is used here because this only does the model selection by default when a model complexity is not chosen